src/Nethermind/Nethermind.TxPool/CompareTxByGasPrice.cs

```diff
@@ -37,11 +37,7 @@ public int Compare(Transaction x, Transaction y)
             if (ReferenceEquals(null, x)) return -1;
                 
             // then by gas price descending
-            int gasPriceComparison = y.GasPrice.CompareTo(x.GasPrice);
-            if (gasPriceComparison != 0) return gasPriceComparison;
-                
-            // then by gas limit ascending
-            return x.GasLimit.CompareTo(y.GasLimit);
+            return y.GasPrice.CompareTo(x.GasPrice);
```

Comment:
this may use a very slow default comparer for structs - can you check?

Reply:
No strucks, just method implementet on UInt256